example/app/data/DemoDataSource/apps/using-remote-package/remoteExample.json

```diff
@@ -0,0 +1,6 @@
+{
+  "_id": "f5282220-4a90-4d02-8f34-b82255fc91d3",
+  "type": "./blueprints/RemoteExample",
+  "name": "remoteExample",
+  "description": ""
```

Comment:
is the "remoteBlueprint" attribute missing? Looks like it is a required attribute in the RemoteExample blueprint